src/Uno.UWP/Networking/Connectivity/NetworkInformation.Android.cs

```diff
@@ -0,0 +1,99 @@
+﻿#if __ANDROID__
+using System.Collections.Generic;
+using System.Linq;
+using Android.App;
+using Android.Content;
+using Uno.Networking.Connectivity.Internal;
+
+namespace Windows.Networking.Connectivity
+{
+	public partial class NetworkInformation
+	{
+		private static ConnectivityChangeBroadcastReceiver _connectivityChangeBroadcastReceiver;
+
+		private static void StartNetworkStatusChanged()
+		{
+			_connectivityChangeBroadcastReceiver = new ConnectivityChangeBroadcastReceiver();
+
+#pragma warning disable CS0618 // Type or member is obsolete

```

Comment:
What's the replacement member for this ?

Reply:
It is sorta-deprecated - https://stackoverflow.com/a/36448021/732221 - apps running in foreground can still use this API, the deprecation is meant for apps listening to network changes in the background.

Reply:
Oh, this is important. Because it could mean that I cannot use this methods on TimeTrigger and similar (#2655), invocation from Toast (#2585) and from LiveTile (#3147) :)

Reply:
@pkar70 hmm, I'm not sure if it is possible to get network status notifications without being in the foreground anymore. It is possible to check for connectivity from the background though.

Reply:
"anymore" in Android, or in UWP? :)

Reply:
Meant Android, at least based on what I found. But maybe I searched wrong :-D . Let me know if you know of a way

Reply:
Now also contains implementation with `NetworkCallback` for newer versions of Android